api/routes/bonds.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require_relative '../../models/wards'
+class App < Roda
+  route('bonds') do |r|
+    # route[List_bonds]: /api/bonds
+    r.get do
+      account_id = rodauth.jwt_session_hash[:account_id]
+      DB[:bonds].join(:schedules, id: :f_schedule_id).join(:wards, id: :f_ward_id).where(f_guardian_id: account_id).map do |v|
+        {
+          'message' => v[:feedback_message],
```

Comment:
Here we have access to columns of the three joined tables, so can present them the way we want

Reply:
Yeah its not so great.
But I suggest we fix that in another PR because so we have more focus when creating the necessary migration.

Reply:
I turned it into an issue: https://github.com/TheRealWolkenbruch/are-you-there-api/issues/24